src/Bridge/Symfony/Bundle/Resources/config/metadata/metadata.xml

```diff
@@ -49,11 +49,6 @@
             <argument type="service" id="api_platform.property_info" />
         </service>
 
-        <service id="api_platform.metadata.property.name_collection_factory.inherited" class="ApiPlatform\Core\Metadata\Property\Factory\InheritedPropertyNameCollectionFactory" decorates="api_platform.metadata.property.name_collection_factory" decoration-priority="10" public="false">
-            <argument type="service" id="api_platform.metadata.resource.name_collection_factory" />
-            <argument type="service" id="api_platform.metadata.property.name_collection_factory.inherited.inner" />
-        </service>
```

Comment:
shouldn't these services be marked as deprecated instead ?

Reply:
This service was only decorating another service. It was never intended to be used directly.